assets/i18n/en_US.json

```diff
@@ -130,7 +131,10 @@
     },
     "patchItem": {
         "unsupportedDialogText": "Selecting this patch may result in patching errors.\n\nApp version: {packageVersion}\nSupported versions:\n{supportedVersions}",
-        "unsupportedPatchVersion": "Patch is not supported for this app version. Enable the experimental toggle in settings to proceed."
+        "unsupportedPatchVersion": "Patch is not supported for this app version. Enable the experimental toggle in settings to proceed.",
+
+        "newPatchDialogText": "This is a new patch.",
```

Comment:
```suggestion
        "newPatchDialogText": "This is a new patch that has been added since the last time you visited the patch selection screen.",
```

Reply:
How about "...since the last time you patched the app" ?

Reply:
Because that's the case.

Reply:
```suggestion
        "newPatchDialogText": "This patch has been added during the time you have last patched.",
```